src/SFA.DAS.AssessorService.Application/Handlers/EpaOrganisationHandlers/CreateEpaOrganisationHandler.cs

```diff
@@ -71,7 +71,8 @@ private static EpaOrganisation MapOrganisationRequestToOrganisation(CreateEpaOrg
                     Address4 = request.Address4,
                     LegalName = request.LegalName,
                     Postcode = request.Postcode,
-                    WebsiteLink = request.WebsiteLink
+                    WebsiteLink = request.WebsiteLink,
+
```

Comment:
Is this intentional not to include  CompanyNumber & CharityNumber

Reply:
good point, now fixed